docs/docusaurus/docs/tutorials/quickstart/quickstart.md

```diff
@@ -179,20 +165,12 @@ To access GX Cloud, you need to join our Cloud Early Access program. During this
 
 ## Next Steps 
 
-Now that you've seen how easy it is to implement the GX workflow, it is time to customize that workflow to suit your specific use cases! To help with this we have prepared some more detailed installation and setting up guides tailored to specific environments and resources.
-
-For more details on installing GX for use with local filesystems, please see:
-
-<SetupAndInstallForFilesystemData />
-
-For guides on installing GX for use with cloud storage systems, please reference:
-
-<SetupAndInstallForCloudData />
+Now that you've seen how easy it is to implement the GX workflow, it is time to customize that workflow to suit your specific use cases! To help with this we have prepared some more detailed guides on setting up and installing GX and getting an initial Data Context that are tailored to specific environments and resources.
 
-For information on installing GX for use with SQL databases, see:
+### Installing GX for specific environments and source data systems
 
-<SetupAndInstallForSqlData />
+<SetupAndInstallGx />
 
-And for instructions on installing GX for use with hosted data systems, read:
+### Initializing, instantiating, and saving a Data Context
```

Comment:
I really like the Next Steps section overall, but it still feels like a tough gap to bridge for the user to persist an ephemeral context. That doc is the last one in the list, and we don't mention Ephemeral Data Context anywhere in Quickstart, so for a user it would not be clear how to go from what they just did to something more persistent unless they start all over again.